.github/workflows/codespell.yml

```
@@ -0,0 +1,22 @@
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

Perhaps we should run this workflow on other branches too?:
```suggestion
on: [push, pull_request]
```
(for example, it could be useful when someone targets the 8.14 branch directly)